src/rating-calculator/components/ChartRecordRow.tsx

```diff
@@ -13,6 +13,7 @@ const SCORE_RECORD_CELL_CLASSNAMES = [
   ACHV_CELL_CLASSNAME,
   'rankCell',
   'ratingCell',
+  'newRatingCell',
```

Comment:
Sorry, I just refactored this file to allow setting CSS class based on ColumnType. Can you pull and merge the latest commit to your branch?

Reply:
No longer required

Reply:
I wonder it is better to use separate CSS classes for ColumnType.RATING and ColumnType.NEXT_RATING. ColumnType.RATING will stay at `width: 70px` while ColumnType.NEXT_RATING is `width: 95px`

Reply:
We can also set `text-align: left` on the `.nextRatingCell` to align the absolute numbers.

Reply:
Had a thought on this - need to test scenario where upscore is > 100. I think 95px will not be enough

Reply:
Reverted rating back to 70px
<img width="1541" alt="Screenshot 2024-03-23 at 8 23 48 PM" src="https://github.com/myjian/mai-tools/assets/162117433/d1000ae2-1c98-4af3-b612-51cf0736a01e">

Centered Rating header with left aligned rows
<img width="1618" alt="Screenshot 2024-03-23 at 8 55 00 PM" src="https://github.com/myjian/mai-tools/assets/162117433/67be6b2b-9ba8-4668-8047-3468dea2e40e">

Updated New Rating column to be 105px, for worst case of upscore of 3 digit rating
<img width="805" alt="Screenshot 2024-03-23 at 8 56 02 PM" src="https://github.com/myjian/mai-tools/assets/162117433/f639f82f-4891-4fe5-a33c-d2ac2bac0ef6">